packages/atomic/src/components/common/result-list/grid-display-results.tsx

```diff
@@ -14,6 +14,17 @@ export const GridDisplayResults: FunctionalComponent<ResultListDisplayProps> = (
       <div
         part="result-list-grid-clickable-container outline"
         ref={(element) => props.setNewResultRef(element!, index)}
+        onClick={(event) => {
+          event.preventDefault();
+          interactiveResult.select();
+          window.open(
+            unfoldedResult.clickUri,
+            event.ctrlKey || event.metaKey
+              ? '_blank'
+              : props.gridCellLinkTarget,
+            'noopener'
+          );
+        }}
       >
         <LinkWithResultAnalytics
```

Comment:
I think this link should also contain the target and `noopener`, but it's not a strong preference.

Reply:
I don't see why not 🙂 I'll quickly add it